docs/dietpi_tools.md

```diff
@@ -80,10 +80,15 @@ You can return to the **DietPi-Software** tool to make further changes at any ti
 
 === "Log System"
 
-    ***\[T.b.d.: Add log system description here\]***
+    Various logging methods can be selected from lightweight to full.
+    If you don’t require log files, get a performance boost. If you need full system logging features, DietPi can do that too.
```

Comment:
Let's change "log files" to "system log files", or "persistent log files" since other applications still log to `/var/log`, only it is cleared once an hour.

Reply:
I will change this to system log files.